Amigolden/src/app/app-routing.module.ts

```diff
@@ -1,89 +1,100 @@
-    path: '',
-    redirectTo: 'home',
-    pathMatch: 'full'
+    path: "",
+    redirectTo: "home",
+    pathMatch: "full",
+  },
+  {
+    path: "home",
+    loadChildren: () =>
+      import("./home/home.module").then((m) => m.HomePageModule),
   },
   {
-    path: 'home',
-    loadChildren: () => import('./home/home.module').then(m => m.HomePageModule)
+    path: RouteNames.login,
+    loadChildren: "./pages/login/login.module#LoginPageModule",
   },
-  { path: RouteNames.login, loadChildren: './pages/login/login.module#LoginPageModule' },
   {
     path: RouteNames.users,
-    loadChildren: './pages/users-list/users-list.module#UsersListPageModule',
-    canActivate: [AuthGuard]
+    loadChildren: "./pages/users-list/users-list.module#UsersListPageModule",
+    canActivate: [AuthGuard],
   },
   {
     path: RouteNames.userDetailFormat,
-    loadChildren: './pages/user-detail/user-detail.module#UserDetailPageModule',
+    loadChildren: "./pages/user-detail/user-detail.module#UserDetailPageModule",
     canActivate: [AuthGuard],
   },
   {
     path: RouteNames.userEvents,
-    loadChildren: './pages/user-events-list/user-events-list.module#UserEventsListPageModule',
-    canActivate: [AuthGuard]
+    loadChildren:
+      "./pages/user-events-list/user-events-list.module#UserEventsListPageModule",
+    canActivate: [AuthGuard],
   },
   {
     path: RouteNames.events,
-    loadChildren: './pages/events-list/events-list.module#EventsListPageModule',
-    canActivate: [AuthGuard]
+    loadChildren: "./pages/events-list/events-list.module#EventsListPageModule",
+    // canActivate: [AuthGuard]
   },
   {
     path: RouteNames.eventDetailFormat,
-    loadChildren: './pages/event-detail/event-detail.module#EventDetailPageModule',
-    canActivate: [AuthGuard]
+    loadChildren:
+      "./pages/event-detail/event-detail.module#EventDetailPageModule",
+    canActivate: [AuthGuard],
   },
   {
     path: RouteNames.enrollmentDetailFormat,
-    loadChildren: './pages/enrollment/enrollment.module#EnrollmentPageModule',
-    canActivate: [AuthGuard]
+    loadChildren: "./pages/enrollment/enrollment.module#EnrollmentPageModule",
+    canActivate: [AuthGuard],
   },
   {
     path: RouteNames.messages,
-    loadChildren: './pages/messages-list/messages-list.module#MessagesListPageModule',
-    canActivate: [AuthGuard]
+    loadChildren:
+      "./pages/messages-list/messages-list.module#MessagesListPageModule",
+    canActivate: [AuthGuard],
   },
   {
     path: RouteNames.messageDetailFormat,
-    loadChildren: './pages/messages/messages.module#MessagesPageModule',
-    canActivate: [AuthGuard]
+    loadChildren: "./pages/messages/messages.module#MessagesPageModule",
+    canActivate: [AuthGuard],
   },
 ];
 
 @NgModule({
   imports: [
-    RouterModule.forRoot(routes, { preloadingStrategy: PreloadAllModules })
+    RouterModule.forRoot(routes, { preloadingStrategy: PreloadAllModules }),
   ],
-  exports: [RouterModule]
+  exports: [RouterModule],
```

Comment:
This file should not be changed.

To avoid changes like this make sure you're using the same conventions in the Ide that is in the TSLint.json